library/NotificationCenter/AutoSuggester.php

```diff
@@ -69,7 +69,11 @@ public function init(\DataContainer $dc)
         $arrParsedTokens = array();
 
         foreach ($arrTokens as $strToken) {
-            $content = $GLOBALS['TL_LANG']['NOTIFICATION_CENTER_TOKEN'][static::$strType][$strToken] ?: '';
+            $content = '';
+
+            if (isset($GLOBALS['TL_LANG']['NOTIFICATION_CENTER_TOKEN'][static::$strType][$strToken])) {
+                $content = $GLOBALS['TL_LANG']['NOTIFICATION_CENTER_TOKEN'][static::$strType][$strToken] ?: '';
+            }
```

Comment:
```suggestion
            $content = $GLOBALS['TL_LANG']['NOTIFICATION_CENTER_TOKEN'][static::$strType][$strToken] ?? '';
```
Should be sufficient, probably.

Reply:
Unfortunately, there is still support for `php ^5.4` in `composer.json` - this is not compatible - shall we update the `php` requirement?

Reply:
Oh right, I did not notice. Yes, increase the version requirement. Supporting PHP 5 is not necessary at all anymore in my opinion.